I created the series and topic models in separate code which can be found [here](###add link###)

First let's look through the series I made...

In [1]:
import os
os.chdir(r'C:\Users\Patrick Stoyer\Dropbox\Documents\Sophomore Year\Semester_2\Ling_1340_Data_Science_for_Linguists\Sentiment-Analysis-of-Russian-Language-News\pickles')
#change to appropriate directory

In [22]:
#unpickle my pickled files
import pickle
pickles = []
for x in os.listdir():
    with open(x,'rb') as f:
        pickles.append(pickle.load(f))
        

In [26]:
#makes seperate things for each pickle
all_lemmas = pickles[0]
bbc_lemmas = pickles[1]
kom_lemmas = pickles[2]
negative = pickles[3]
positive = pickles[4]
rad_lemmas = pickles[5]
reut_lemmas = pickles[6]
tas_lemmas = pickles[7]

In [37]:
#below are the types and lengths by the way
for x in pickles:
    print (type(x))
    print(len(x))



<class 'pandas.core.series.Series'>
6751
<class 'pandas.core.series.Series'>
351
<class 'pandas.core.series.Series'>
5293
<class 'pandas.core.frame.DataFrame'>
499
<class 'pandas.core.frame.DataFrame'>
500
<class 'pandas.core.series.Series'>
3056
<class 'pandas.core.series.Series'>
286
<class 'pandas.core.series.Series'>
3058


In [32]:
#here is a sample, the third one is probably a site that didn't have anything to scrape (maybe a video or gallery)
#you may not be able to tell but all words are in the base forms (i.e. lemmatized). I still have to go back and alter it slightly, 
#because some punctuation is not removed e.g. in 70
kom_lemmas[:100]

0     [сегодня, столица, сторонник, кпрф, левый, пар...
1     [группа, 16, страна, организация, безопасность...
2     [верховный, суд, коми, вынести, приговор, резо...
3                                                    []
4     [компания, quacquarelli, symonds, qs, опублико...
5     [ученый, северо-западный, университет, чикаго,...
6     [международный, аэропорт, сочи, задерживаться,...
7     [международный, правозащитный, организация, am...
8     [международный, правозащитный, организация, am...
9     [основатель, baring, vostok, майкл, калви, аре...
10    [издание, the, insider, расследовательский, гр...
11    [участник, pussy, riot, издатель,  «, медиазон...
12    [эксперт, bloomberg, подсчитывать, российский,...
13    [аварийный, посадка, boeing, 737, сочи, 18, че...
14    [россия, проводить, успешный, испытание, гипер...
15    [платформа, coursera, проанализировать, активн...
16    [доля, 10, самый, обеспечивать, россиянин, при...
17    [ъ, продолжать, серия, репортаж, регион, д

In [40]:
#I should have made this a series, but it basically just has words wit negative emotions like подавленный, depressed, or угрюмый
#gloomy.
negative[:100]
#i'm not sure how I'll use this... i tried to find all lda models that have any of these words but there weren't any.

0
0                 злобно
1              волновать
2              устрашать
3           запуганность
4            неприличный
5           раздражающий
6            застенчивый
7               алчность
8         подозрительный
9                   тень
10                  себя
11           подавленный
12          запаниковать
13               одиозно
14               угрюмый
15            обуздывать
16        замешательство
17             надоедать
18               грязный
19             обижаться
20            содрогание
21            паникующий
22        безжалостность
23             презирать
24            выходом_на
25            усугублять
26               бояться
27                досада
28           убийственно
29  женоненавистничество
..                   ...
70           негодование
71               гневить
72          отвращение_к
73              докучать
74           безжалостно
75              противно
76             ненависть
77        воинственность
78              вопиющий
79  классовый_антагонизм
80          тревожащийся
81          нерешительно
82              напугать
83  детоненавистничество
84             чудовищно
85                унылый
86         злопамятность
87             женофобия
88           тропа_войны
89             неприятно
90          пренебрегать
91            неприятный
92          отталкивающе
93           недовольный
94              задевать
95              испугать
96                 лютый
97            застенчиво
98         неуверенность
99                 сцену

[100 rows x 1 columns]

Now lets check out the lda and lsa models (i made them with gensim btw)...

In [41]:
os.chdir(r'C:\Users\Patrick Stoyer\Dropbox\Documents\Sophomore Year\Semester_2\Ling_1340_Data_Science_for_Linguists\Sentiment-Analysis-of-Russian-Language-News\gensim')
#change to appropriate directory)


In [42]:
from gensim.models.ldamodel import LdaModel
#import LdaModel

In [43]:
lda_bbc = LdaModel.load("ldamodel_all.gensim")

In [44]:
lda_bbc.print_topics(20)
#i probably should have downloaded python's google translate api, I'll do that later but for now I'll just translate this 
#"manually" through google translate 

[(132,
  '0.019*"дом" + 0.016*"строительство" + 0.011*"объект" + 0.010*"компания" + 0.009*"квартира" + 0.009*"дольщик" + 0.008*"жилой" + 0.007*"жилье" + 0.007*"город" + 0.007*"проблемный"'),
 (25,
  '0.009*"государственный" + 0.008*"университет" + 0.006*"воронеж" + 0.005*"россия" + 0.005*"член" + 0.005*"палач" + 0.005*"совет" + 0.005*"становиться" + 0.004*"общественный" + 0.004*"сентябрь"'),
 (75,
  '0.033*"ингушетия" + 0.033*"чечня" + 0.024*"республика" + 0.020*"кадыров" + 0.019*"глава" + 0.019*"граница" + 0.012*"суд" + 0.011*"соглашение" + 0.010*"рамзан" + 0.009*"конституционный"'),
 (80,
  '0.021*"федотов" + 0.019*"ливия" + 0.014*"хафтар" + 0.013*"сергеев" + 0.012*"ливийский" + 0.010*"москва" + 0.010*" – " + 0.009*"италия" + 0.007*"алексиевич" + 0.007*"поставка"'),
 (188,
  '0.016*"золото" + 0.014*"пожар" + 0.013*"место" + 0.012*"торговый" + 0.012*"коррупция" + 0.012*"россия" + 0.011*"рейтинг" + 0.010*"балл" + 0.009*"страна" + 0.009*"занимать"'),
 (57,
  '0.016*"владимирский" + 0.00

Btw the descriptions are my own not from the program

1. (132,
  '0.019 * "house" + 0.016 * "construction" + 0.011 * "object" + 0.010 * "company" + 0.009 * "apartment" + 0.009 * "equity holder" + 0.008 * "residential" + 0.007 * "housing" + 0.007 * "city" + 0.007 * "problematic" ')
  
      - Maybe about new housing, possibly there is a housing shortage somewhere.
  
  
2. (25,
  '0.009 * "state" + 0.008 * "university" + 0.006 * "Voronezh" + 0.005 * "Russia" + 0.005 * "member" + 0.005 * "executioner" + 0.005 * "advice" + 0.005 * "become" 0.004 * "public" + 0.004 * "September" ')
  
    - Maybe this is about some sort of case where a member of a Voronezh university was executed or something, but I am unfamiliar with this
    
3. (75,
  '0.033 * "Ingushetia" + 0.033 * "Chechnya" + 0.024 * "Republic" + 0.020 * "Kadyrov" + 0.019 * "Chapter" + 0.019 * "Border" + 0.012 * "Court" + 0.011 * "Agreement" + 0.010 * "ramzan" + 0.009 * "constitutional" '),
  
    - This seems to relate to the Caucasus, the first few terms are regions, Kadyrov is an official in Chechnya etc. Maybe some sort of border agreement was passed
 
4. (80,
  '0.021 * "Fedotov" + 0.019 * "Libya" + 0.014 * "Haftar" + 0.013 * "Sergeev" + 0.012 * "Libyan" + 0.010 * "Moscow" + 0.010 * "-" + 0.009 * "Italy" + 0.007 * "Aleksiyevich" + 0.007 * "Delivery" ')
    
    - Not sure who these people are
 
5. (188,
  '0.016 * "gold" + 0.014 * "fire" + 0.013 * "place" + 0.012 * "trade" + 0.012 * "corruption" + 0.012 * "Russia" + 0.011 * "rating" + 0.010 * "score" + 0.009 * "country" + 0.009 * "occupy" '),
  
    - gold, place, trade and corruption all seem pretty related, not sure what fires have to do with that though

 
 6. (57,
  '0.016 * "Vladimir" + 0.007 * "Russia" + 0.007 * "Tass" + 0.006 * "nomination" + 0.005 * "Book" + 0.005 * "Award" + 0.005 * "Stringino" + 0.005 * "First" + 0.005 * "-" + 0.005 * "Glinka" '),
 
     - some kind of award, not sure what...
 
7.  (40,
  '0.072 * "stadium" + 0.038 * "football" + 0.035 * "match" + 0.033 * "arena" + 0.020 * "world" + 0.018 * "hm" + 0.012 * "fifa" + 0.010 * "Russia" + 0.009 * "April" + 0.008 * "", "'),
  
      - pretty obviously about the world cup
 
8. (0,
  '0.041 * "Vladivostok" + 0.027 * "East" + 0.026 * "Far" + 0.019 * "Primorsky" + 0.019 * "Federal" + 0.017 * "Far Eastern" + 0.017 * "Primorye" + 0.016 * "District" + 0.015 * "edge" + 0.014 * "tass" '),
     
     - all terms related to the far west
 
 
9. (four,
  '0.024 * "reserve" + 0.016 * "arctic" + 0.015 * "territory" + 0.010 * "natural" + 0.008 * "state" + 0.008 * "", - "+ 0.008 *" arctic "+ 0.008 *" area " + 0.007 * "", "+ 0.007 *" Caucasian "'),
  
      - some kind of new reserve in the artic region
  
10. (3,
  '0.025 * "School" + 0.022 * "Change" + 0.018 * "Yakutsk" + 0.011 * "Training" + 0.010 * "Buryatia" + 0.009 * "", - "+ 0.009 *" Child "+ 0.009 *" Tass " + 0.009 * "Republic" + 0.008 * "Yakut" '),
      
      - not sure exactly, but obviously about schooling
  
11. (176,
  '0.018 * "Russia" + 0.012 * "Nuclear" + 0.009 * "New" + 0.009 * "Titorenko" + 0.008 * "Novel" + 0.008 * "Repertory" + 0.008 * "Author's" + 0.008 * "Russian" + 0.008 * "d" + 0.007 * "l" ')
  
    - Not sure, but vaguely scary
 
12. (179,
  '0.016 * "Estonia" + 0.013 * "Russia" + 0.009 * "screen" + 0.009 * "Estonian" + 0.008 * "Helm" + 0.007 * "Party" + 0.005 * "Estonian" + 0.005 * "People" + 0.005 * "", - "+ 0.005 *" reserve "')
  
      - about estonia, maybe with more terms this would be more meaningful
 
13. (117,
  '0.020 * "Ruble" + 0.009 * "Russia" + 0.009 * "City" + 0.008 * "Thousand" + 0.007 * "Day" + 0.007 * "Bondarenko" + 0.007 * "Hanseatic" + 0.007 * "Declare" + 0.006 * "Hanse" + 0.005 * "Horse" '),
  
      - hanseatic is probably referring to the hanseatic league... maybe there's some sort of new holiday memorializing it althought that seems weird. (it seems there was a [new documentary](https://tvkultura.ru/brand/show/brand_id/63425/) about the hanseatic league last year.
 
14. (120,
  '0.012 * "police" + 0.010 * "nightingales" + 0.010 * "shitik" + 0.008 * "island" + 0.007 * "Kronstadt" + 0.006 * "activist" + 0.006 * "Egypt" + 0.006 * "presenter" + 0.006 * "John" + 0.005 * "word" '),
  
      - not sure the exact context, but these seem reasonable (nightingales shouldn't have been translated, it is a person (solovyov)
 
15. (39,
  '0.011 * "project" + 0.006 * "Russia" + 0.005 * "thunder" + 0.004 * "Russian" + 0.004 * "work" + 0.004 * "word" + 0.004 * "Mikhail" + 0.004 * "new" + 0.004 * "ruble" + 0.003 * "part" ')
      
      - about a new project, but need more terms to know anything more specific
 
16. (7,
  '0.048 * "airport" + 0.041 * "flight" + 0.033 * "airplane" + 0.026 * "airline" + 0.020 * "passenger" + 0.019 * "tass" + 0.017 * "Moscow" + 0.010 * "board" + 0.009 * “fly out” + 0.008 * "", - "'),
  
     - obvious
     
17. (64,
  '0.016 * "vodka" + 0.013 * "ocean" + 0.011 * "shabalin" + 0.010 * "museum" + 0.008 * "project" + 0.008 * "Vladimir" + 0.007 * "first" + 0.006 * "white" + 0.006 * "planet" + 0.006 * "smirnovs" ')
  
      - is there a vodka musuem??
 
18. (100,
  '0.012 * "sixun" + 0.010 * "project" + 0.006 * "district" + 0.006 * "deputy" + 0.006 * "person" + 0.006 * "Malenko" + 0.006 * "pension" + 0.005 * "area" + 0.005 * "", - "+ 0.005 *" tass "'),
  
      - all these make sense, but not sure what they refer to
 
19. (198,
  '0.036 * "court" + 0.015 * "lawyer" + 0.011 * "case" + 0.010 * "admit" + 0.010 * "group" + 0.010 * "Russia" + 0.009 *  "concert" + 0.009 * "radio" + 0.009 * "svoboda" + 0.009 * "america" ​​'),
  
     - about a court case, maybe one covered by or involving radio svoboda
20. (134,
  '0.056 * "Technopark" + 0.039 * "Kvantorium" + 0.037 * "Equipment" + 0.028 * "Children's" + 0.026 * "Child" + 0.016 * "Gas" + 0.013 * "Education" + 0.012 * "Additional" + 0.009 * "high-tech" + 0.009 * "engineering" ')
      
      - not sure what these are but all them make sense together
      
As you can see, most of the topics are pretty reasonable, even if they are vague. Btw, iirc print_topic(N) prints the N most significant topics or something like that. For all of these, I have written their outputs for 20 topics to files which can be found [here](###asjdf###)